app/javascript/components/rules/index.js

```diff
@@ -52,56 +67,69 @@ class Rule extends React.Component {
           ]}
         >
           <p>
-            { status }
+            <RuleStatus rule={rule}/>
           </p>
         </Card.Section>
       </Card>
     );
   }
 }
 
+class SectionEnabled extends React.Component {
+  render () {
+    return(
+      <Layout.AnnotatedSection
+        id="ruleIndex"
+        title="Enabled rules"
+        description="Rules are listening to events produced by your Shopify store."
+      >
+        {
+          this.props.rules.map(function(rule, index) {
+            return <Rule rule={ rule } key={ index } />
+          })
+        }
+      </Layout.AnnotatedSection>
+    )
+  }
+}
+
+class SectionDisabled extends React.Component {
+  render () {
+    if (this.props.rules.length === 0) {
+      return (
+        <></>
+      )
+    }
```

Comment:
I'm not sure it make sense for a component to know not to render itself. Maybe the conditional goes in the parent. 🤷

Reply:
Not sure either. You do you.